Sources/StreamChatUI/ChatMessageList/ChatChannelVC.swift

```diff
@@ -25,8 +25,10 @@ open class _ChatChannelVC<ExtraData: ExtraDataTypes>: _ChatVC<ExtraData> {
         _ handler: @escaping (ChatChannelNavigationBarListener<ExtraData>.NavbarData) -> Void
     ) -> ChatChannelNavigationBarListener<ExtraData>? {
         guard let channel = channelController.channel else { return nil }
-        let namer = uiConfig.messageList.channelNamer.init()
-        let navbarListener = ChatChannelNavigationBarListener.make(for: channel.cid, in: channelController.client, using: namer)
+        let navbarListener = ChatChannelNavigationBarListener.make(
+            for: channel.cid,
+            in: channelController.client,
+            using: DefaultChatChannelNamer())
```

Comment:
Same here, the namer must be configurable, otherwise - why to even have it?

Reply:
Yes, that's true. Fixed.